src/gluonts/model/estimator.py

```diff
@@ -40,7 +40,10 @@ def __init__(self, lead_time: int = 0, **kwargs) -> None:
         self.lead_time = lead_time
 
     def train(
-        self, training_data: Dataset, validation_data: Optional[Dataset] = None
+        self,
+        training_data: Dataset,
+        validation_data: Optional[Dataset] = None,
+        from_predictor: Optional[Predictor] = None,
```

Comment:
Is this a breaking feature, because I have to accept the `from_predictor` argument in subclasses?

Maybe a second method would be better than, so models can opt in to implement it?